crates/partition-store/src/invocation_status_table/mod.rs

```diff
@@ -352,9 +341,9 @@ mod tests {
     fn round_trip() {
         let expected_invocation_id = InvocationId::mock_random();
 
-        let key = create_invocation_status_key(&expected_invocation_id).serialize();
+        let key = create_invocation_status_key_v1(&expected_invocation_id).serialize();
 
-        let actual_invocation_id = invocation_id_from_old_key_bytes(&mut key.freeze()).unwrap();
+        let actual_invocation_id = invocation_id_from_v1_key_bytes(&mut key.freeze()).unwrap();
```

Comment:
Should this now try to serialize and deserialize the new invocation status (V2)?